megamek/src/megamek/server/Server.java

```diff
@@ -23913,6 +23053,8 @@ private void resolveShutdownCrashes() {
                             / game.getOptions().intOption(
                                     OptionsConstants.ADVCOMBAT_VEHICLES_THRESHOLD_DIVISOR));
 
+                    // TODO : Windchild fix integer division in floating point context
```

Comment:
I'm pretty sure this should be fixed, as it is integer division in floating point context. (can't remember why I wrote it as a comment, probably because it is a variable option)